src/cljs/ui/events/graphql.cljs

```diff
@@ -338,6 +338,15 @@
            (dissoc-in [:users :authorized-user])
            (dissoc-in [:users user-id]))})
 
+(defmethod handler :send-login-email
+  [_ _ _]
+  ;; TODO : create subscription for email status (when its implemented on the API side)
```

Comment:
Is this intentional?

Reply:
yes, sendgrid has a hook for if email arrives, so I though we could subscribe to it and notify user if email arrives (or fails to), but it's outside the scope of this PR imo